### ISSUE TITLE
[HU04] Como usuario quiero poder cambiar mi contraseña

### ISSUE THREAD

**Issue by Torchu** [Torchu/flixbuff#22]
- [ ] Crear un sistema de recuperación de contraseña

**Comment by JJ** [comment]
No, ¿por qué? ¿En qué PMV va esto? No es una historia de usuario, ni tienes claro desde el principio que vayas a necesitar ese tipo de cosas. Dependerá de la solución que adoptes. A lo mejor hay un administrador que puede cambiar el password.
Meterlo también en un sistema de reseñas, cuando no tiene nada que ver con el sistema de reseñas, es meter un montón de características en un PMV muy temprano.
Como ya te he dicho por otro lado, céntrate en los modelos de datos y la lógica de negocios. Lo administrativo lo puedes dejar para luego (por ejemplo, usar login de Google). Si no sabes que eso es algo que va a resolver algún problema, no lo pongas.
